Tools/eslint-config-cesium/LICENSE.md

```diff
@@ -0,0 +1,7 @@
+Copyright 2017 Ottavio Hartman, Analytical Graphics, Inc., and Contributors
```

Comment:
You can delete this license, it's covered by the main LICENSE.md at the top of the repository.